src/amber/cli/commands/database.cr

```diff
@@ -52,17 +52,19 @@ module Amber::CLI
             `crystal db/seeds.cr`
             Micrate.logger.info "Seeded database"
           when "migrate"
```

Comment:
We should probably cover all off these commands `migrate,rollback,redo,status,versions` since all of them depend on the directory. No?

Reply:
@eliasjpr my first commit, ensure new apps always has the `db/migrations` directory. Is that enough?